packages/react-widgets/src/layers/EditableCartoGeoJsonLayer.js

```diff
@@ -0,0 +1,30 @@
+import { EditableGeoJsonLayer } from '@nebula.gl/layers';
+
+const EVENT_TYPES = ['anyclick', 'pointermove', 'panstart', 'panmove', 'panend', 'keyup'];
```

Comment:
Probably a comment with a bit of explanation on this would be really useful, about how this should be better a mechanism in nebula itself, with probably a PR on that in the future

Reply:
Agree, I'm going to add a link to the original code and write a small explanation

Reply:
Done